### DISCUSSION TITLE
Which IDE?

### DISCUSSION THREAD

**Discussion by mghildiy** [grafana/tempo#2292] [General]
Hello All,

I am trying to open repo in Jetbrains' goland IDE, but it freezes everytime.
So wanted to know which IDE is considered to provide with best experience for golang development?

**Comment by mapno** [comment]
Hello @mghildiy.

I haven't heard of issues opening Tempo before. I use Goland daily and rarely encounter any problems. Unsure what the problem can be. I'd try opening the project in a more lightweight IDE.

**Reply by mghildiy** [reply]
You are right, works fine with VSC.
